app/assets/javascripts/direct_uploads.js

```diff
@@ -2,6 +2,8 @@
 /* original source:
```

Comment:
Is this comment still accurate?

Reply:
yes, we've made a line or two of tweaks but it's largely copy/pasta from those docs. 👍